src/rust/lib/debug-scenes/src/shapes.rs

```diff
@@ -181,6 +181,8 @@ use graph_editor::component::node::port::Expression;
 
 
 pub fn expression_mock() -> Expression {
+    let pattern_cr       = vec![Seq { right: false }, Or, Or, Build];
```

Comment:
`cr` is not really a descriptive name
(and the situation here doesn't really justify us using shortcuts)